### DISCUSSION TITLE
your plugin is not working with last testrail api

### DISCUSSION THREAD

**Discussion by hatufacci** [PeterNgTr/codeceptjs-testrail#64] [General]
your plugin is not working with last testrail api!
when you plan to fix it and add https://github.com/PeterNgTr/codeceptjs-testrail/pull/59 ?
it was ready almost 3 month ago - and issue still persist @PeterNgTr

**Comment by hatufacci** [comment]
1.8.12 works perfectly!

**Comment by kobenguyent** [comment]
Just tested with latest Testrail cloud based. No issue.
